torch/onnx/symbolic_opset9.py

```diff
@@ -1950,15 +1944,47 @@ def multinomial(g, input, num_samples, replacement=False, generator=None):
                 dtype_i=sym_help.cast_pytorch_to_onnx['Long'],
                 sample_size_i=num_samples)
 
+
 def baddbmm(g, self, batch1, batch2, beta, alpha):  
     dtype = self.type().scalarType()    
     batch_mul = matmul(g, batch1, batch2)       
     mul_a = mul(g, batch_mul, g.op("Cast", alpha, to_i=sym_help.cast_pytorch_to_onnx[dtype]))       
     mul_b = mul(g, self, g.op("Cast", beta, to_i=sym_help.cast_pytorch_to_onnx[dtype]))
     return add(g, mul_a, mul_b)
 
+
 def gelu(g, self):
     _sqrt2 = 1.4142135623730951
     erf = g.op('Erf', div(g, self, torch.tensor(_sqrt2)))
     erf_plusone = add(g, erf, g.op('Constant', value_t=torch.tensor(1, dtype=torch.float)))
     return mul(g, mul(g, self, erf_plusone), g.op('Constant', value_t=torch.tensor(0.5, dtype=torch.float)))
+
+
+@parse_args('v', 'i', 'v', 'v', 'f', 'i')
+def group_norm(g, input, num_groups, weight, bias, eps, cudnn_enabled):
+    input_sizes = input.type().sizes()
```

Comment:
This requires input shape to be known. Consider adding a helper function in symbolic_helper.py, since this pattern appears in a lot of places.

Reply:
I'm adding a check for if input.isCompleteTensor()
I can add a helper function which does this check internally, but just returning input sizes is not that helpful

Reply:
I think I can send this out as a new PR, don't really want to touch other ops in this PS